Source/JavaScriptCore/runtime/JSDataViewPrototype.cpp

```diff
@@ -159,12 +159,15 @@ EncodedJSValue getData(JSGlobalObject* globalObject, CallFrame* callFrame)
         return throwVMTypeError(globalObject, scope, typedArrayBufferHasBeenDetachedErrorMessage);
 
     size_t byteLength = byteLengthValue.value();
-    if (elementSize > byteLength || byteOffset > byteLength - elementSize)
+    if (byteOffset > std::numeric_limits<size_t>::max()
```

Comment:
Why do we need this check additionally?

Reply:
In 32bit env, `uint64_t` is too large for `size_t`. That's why the range check is added. Is it unnecessary?